pkg/xds/bootstrap/generator.go

```diff
@@ -370,6 +370,15 @@ func (b *bootstrapGenerator) xdsHost(request types.BootstrapRequest) string {
 	}
 }
 
+func (b *bootstrapGenerator) adminAccessLogPath(operatingSystem string) string {
+	if b.config.Params.AdminAccessLogPath == os.DevNull && operatingSystem == "windows" {
```

Comment:
this is nit picking, but I'd add a separate check: 
```go
if operatingSystem == "" {
    return b.config.Params.AdminAccessLogPath
}
```
so it's obvious we're ready to accept request from old clients